asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -333,6 +342,10 @@ def main():
         raise ValueError("Need to specify exactly one of --exp_file or --lig_file.")
     n_mols = len(mols)
 
+    # load ml models
+
+    GAT_model = GATInference("model1")
```

Comment:
I think we should probably make running the prediction optional, eg with some boolean CLI flag or something